### DISCUSSION TITLE
Facebook group

### DISCUSSION THREAD

**Discussion by arcolinuxz** [leftwm/leftwm#544] [Ideas]
created a facebook page

it is yours any time you want

https://www.facebook.com/groups/641534130170154

**Comment by mautamu** [comment]
Should we add this to the README as well?

**Reply by hertg** [reply]
Hmmm, depends. I don't think there's a lot of facebook users in the linux community. But I may be out of the loop, because everyone in my circles ditched fb and its services for good. If there was a community on social media, a leftwm subreddit may be more appropriate?

**Reply by VuiMuich** [reply]
> Hmmm, depends. I don't think there's a lot of facebook users in the linux community. But I may be out of the loop, because everyone in my circles ditched fb and its services for good. If there was a community on social media, a leftwm subreddit may be more appropriate?

I agree a subreddit feels more at home for a Linux social media community then on fb.

**Comment by arcolinuxz** [comment]
heads up
delete the FB group
